client/package.json

```diff
@@ -6,9 +6,9 @@
   },
   "scripts": {
     "clean": "rimraf output",
-    "test": "spago test --path $npm_package_config_configpath",
-    "build": "spago build --path $npm_package_config_configpath",
-    "bundle": "spago bundle-app --path $npm_package_config_configpath --purs-args '--censor-lib --strict' --to public/js/index.js"
+    "test": "spago test --path config/dev/Try.Config.purs",
+    "dev": "spago build --path config/dev/Try.Config.purs",
```

Comment:
If we are going to hardcode this path here, I think we should just hardcode the production config path too. How about having the dev one called `build` and the production one called `build_ci` or `build_production`?